### ISSUE TITLE
Incorporate new data (ENVEO & Landsat Alps)

### ISSUE THREAD

**Issue by marlens123** [RolnickLab/snowgalileo#19]
_(on hold until data is accessible)_
- [x] Landsat: download and upload from terrabyte to Mila cluster
- [x] Landsat: export corresponding input tifs
- [x] Landsat: preprocess data (filename adjustments, cropping, data splits, adjust eval configs)
- [x] Landsat: create train and validation set
- [x] Landsat: test fine-tuning
- [x] Landsat: create pipeline to operate with Alps / Rockies test sets
- [x] Landsat: start hyperparameter tuning
- [ ] Landsat: create experimentation scripts for cloud, forest, and season filtering
- [ ] Landsat: create training data subsets with different amounts (for data ablation)
- [ ] ENVEO: (?) (information missing)

**Comment by marlens123** [comment]
**Update 12/12:**
- Landsat test data available via terrabyte, waiting for folder access
- ENVEO data pending

**Comment by marlens123** [comment]
NOTE: If results are bad --> return to masking out zeros in input data

**Comment by marlens123** [comment]
Note:

- increased the number of patches per FSC bin to 100 (results in ~ 31000 train and test samples)
- created an extra balanced version, with manually removed 3000 FSC0 patches and additionally exported 500 FSC100 patches (excluded in previous version due to code logic)
